<launch>
	<node pkg="robofriend" type="PCBSensorData.py" name="robofriend_pcb_sensor_data" output="screen">
	</node>

	<node pkg="robofriend" type="RobobrainTalkerNode.py" name="robofriend_talker" output="screen">
	</node>
	
</launch>
